src/components/hero.js

```diff
@@ -1,20 +1,26 @@
 import React from 'react';
 
+import bg from '../static/hero.jpg';
```

Comment:
It's better to be explicit. `bg` could be any background and will need refactored when there are more images added using this convention.

```suggestion
import heroImage from '../static/hero.jpg';
```